inflammation/models.py

```diff
@@ -46,3 +46,30 @@ def daily_min(data):
     :returns: An array of minimum values of measurements for each day.
     """
     return np.min(data, axis=0)
+
+def patient_normalise(data):
```

Comment:
```suggestion
def patient_normalise(data):
```

Perhaps it would be better to merge this into the develop branch separately, instead of having both features in this one